### PR TITLE
Match CreateWaterPulseRingBubbles

### PR DESCRIPTION
**Pull request by PokeCodec**

### CODE REVIEW

**Comment by GriffinRichards** [comment]
This was merged incorrectly, it contains duplicates of commits already in the repo

**Comment by PokeCodec** [comment]
Fixed

File: src/battle_anim_water.c [hunk 1]
```diff
@@ -1931,216 +1931,50 @@ static void AnimWaterPulseRing_Step(struct Sprite *sprite)
...
-	pop {r3-r5}\n\
-	mov r8, r3\n\
-	mov r9, r4\n\
-	mov r10, r5\n\
-	pop {r4-r7}\n\
-	pop {r0}\n\
-	bx r0\n");
-}
-#endif
+}
```
Comment (GriffinRichards) [inline src/battle_anim_water.c:1980]: Put the terminal newline back

**Comment by PokeCodec** [comment]
Fixed

**Pull request merged**